src/nile/cli.py

```diff
@@ -183,11 +186,25 @@ async def declare(
 
 @cli.command()
 @click.argument("signer", nargs=1)
+@click.option("--salt", nargs=1)
+@click.option("--max_fee", nargs=1)
 @network_option
 @watch_option
-async def setup(signer, network, watch_mode):
+async def setup(signer, network, salt, max_fee, watch_mode):
```

Comment:
Hi Marto, I was trying to test the setup of an account, and it looks like I can't `setup` an account because the `class_hash` is not declared, and I can't declare it because I have no account deployed to declare through. Am I missing something?